### PR TITLE
Adding in talks & videos

### PR DESCRIPTION
**Pull request by sboundy**
Adding in dev talks & videos

### CODE REVIEW

File: meetups.html [hunk 1]
```diff
@@ -2,14 +2,47 @@
...
 ---
-<div class="skycom-12">
-    <div class="skycom-container clearfix">
-        <div class="alpha skycom-12">
-            <h1></h1>
-            <h3>Hear about what we're doing</h3>
-            <p class="lead text--instructions">Take a look at what's going on for Developers across Sky. Then why not come along to one of our Meetups</p>
-            <h1></h1>
-            <iframe src="https://www.google.com/calendar/embed?showTitle=0&amp;showNav=0&amp;showPrint=0&amp;showTabs=0&amp;showCalendars=0&amp;showTz=0&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=5sbrf4s4spehi2ssq4apchodp4%40group.calendar.google.com&amp;color=%23865A5A&amp;ctz=Europe%2FLondon" style=" border-width:0 " width="1140" height="600" frameborder="0" scrolling="no"></iframe>
+<h1></h1>
```
Comment (cookchristopher) [inline meetups.html]: Should this be empty?

Comment (sboundy) [inline meetups.html]: Nah - it shouldn't.  It needs some padding - I'll tweak the .css and update.

File: _includes/scripts.html [hunk 2]
```diff
@@ -1,5 +1,12 @@
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
 <script src='http:////web-toolkit.global.sky.com/2.3.6/scripts/toolkit.min.js'></script>
```
Comment (joffreytourret) [inline _includes/scripts.html]: Minor thing but the script src to the toolkit has four '/'

**Comment by sboundy** [comment]
Thanks for feedback @joffreytourret and @cookchristopher .  Have updated with a couple of tweaks.  This pull request also includes addition of Google Analytics.

**Comment by andrewmerrell** [comment]
How do we get access to view the ga stats, is it on a sky account somewhere?

**Comment by sboundy** [comment]
Will contact you and set you up @andrewmerrell

**Comment by AndrewJack** [comment]
:+1: LGTM
